### PR TITLE
Added in convos for URL minigame

### PR DESCRIPTION
**Pull request by SelectiveHouse**
Took some liberties with 3.4a

### CODE REVIEW

File: app/minigames/urls/conversations/1-intro.oulipo [hunk 1]
```diff
@@ -13,63 +15,63 @@ Arca:
...
 Arca:
 	- Oh... did he say why he was looking for them?
 		continue
 	- Did you get his name?
 		continue
 	- Do you know where he was going?
 		continue
-		
+
 Hobo [irritated]: No
```
Comment (SomeHats) [inline app/minigames/urls/conversations/1-intro.oulipo]: This line is missing an annotation

File: app/minigames/urls/conversations/4-date-location.oulipo [hunk 2]
```diff
@@ -1,53 +1,55 @@
...
 set view.characters.hobo.name = 'Boris J. Buttstacks'
 
-Hobo [really-happy]: You did jolly well fetching me my teeth! I'm awfully glad Greasy Pete was civil! He can be rather bothersome at times. You're a true friend.
+Hobo [really-happy track:01b-jolly-well]: You did jolly well fetching me my teeth! I'm awfully glad Greasy Pete was civil! He can be rather bothersome at times. You're a true friend.
 go menu
 
 #menu
-Arca: 
+Arca:
 	- SKIP CONVERSATION -> skip
```
Comment (SomeHats) [inline app/minigames/urls/conversations/4-date-location.oulipo]: Could you remove this and any others like it please?

**Comment by SomeHats** [comment]
This looks good! There's a couple comments for you to address. Can you also go through and test that all of these actually work if you haven't already?

**Comment by SomeHats** [comment]
You can start a specific conversation like this:
`eak.startConversation('/minigames/urls/conversations/1-intro')` when you're on the map view

File: app/minigames/urls/conversations/1-intro.oulipo (hunk 1 above)
Comment (SelectiveHouse) [inline app/minigames/urls/conversations/1-intro.oulipo]: Yeah the 04f-didnt-say.mp3 actually says: "No, he didn't say" shall I add in the text?

**Comment by SelectiveHouse** [comment]
Shall we close this branch? Looks like the new one does all the changes (but better! )

**Pull request closed without merging**